lib/components/src/tooltip/WithTooltip.tsx

```diff
@@ -129,26 +128,26 @@ const WithTooltip = lifecycle({
       });
     });
 
-    this.unbind = () => {
+    (this as any).unbind = () => {
       document.removeEventListener('keydown', hide);
       unbinders.forEach(unbind => {
         unbind();
       });
     };
   },
   componentWillUnmount() {
-    if (this.unbind) {
-      this.unbind();
+    if ((this as any).unbind) {
+      (this as any).unbind();
```

Comment:
Why do you needed to cast `this as any` ?

Reply:
```
Error:(139, 16) TS2339: Property 'unbind' does not exist on type '_ReactLifeCycleFunctionsThisArguments<WithTooltipPureProps, {}>'.
```
I think `WithTooltip` component need to be refactored in order to make these `bind` and `unbind` functions class properties or something like that. I keep it in mind for another PR.

Reply:
👍